boudicca.base/query-lib/src/main/kotlin/base/boudicca/query/evaluator/OptimizingEvaluator.kt

```diff
@@ -30,6 +30,9 @@ import java.time.format.DateTimeParseException
 import java.util.BitSet
 import java.util.concurrent.ConcurrentHashMap
 
+private const val SORTING_RATIO = 3
+
+@Suppress("detekt:TooManyFunctions")
```

Comment:
do we really need this rule?

Reply:
i think it normally makes sense to have it, in most cases this is a good indicator that you should split the class